src/commands/run.js

```diff
@@ -28,7 +28,8 @@ export const handler = async (argv) => {
     const accessKey = config.key || process.env.SAUCE_ACCESS_KEY
     const jobName = getJobName(config)
     const buildName = config.build || `${jobName} - ${(new Date()).toString()}`
-    const metrics = getMetricParams(config)
+    const budget = config ? config.budget : null
+    const metrics = budget ? Object.keys(budget) : getMetricParams(config)
```

Comment:
What happens if someone has a spelling error in the metrics? Do we handle the case properly?

Reply:
oh no, I will add validation logic.